File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/response/ResponseModules.java
package app.editors.manager.mvp.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.user.Module;

public class ResponseModules extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private List<Module> response;

    public List<Module> getResponse() {
        return response;
    }

    public void setResponse(List<Module> response) {
        this.response = response;
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseModules.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.manager.models.user.Module
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseModules(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: List<Module>? = null
) : BaseResponse()